src/display/dom_utils.js

```diff
@@ -39,24 +39,29 @@ DOMCanvasFactory.prototype = {
   create: function DOMCanvasFactory_create(width, height) {
     assert(width > 0 && height > 0, 'invalid canvas size');
     var canvas = document.createElement('canvas');
+    var context = canvas.getContext('2d');
     canvas.width = width;
     canvas.height = height;
-    return canvas;
+    return {
+      canvas: canvas,
+      context: context
```

Comment:
Nit: add a trailing comma here, i.e., `context: context,`. This is to avoid unnecessarily line changes in the diff later on if new items are added to this object.